modules/shortcodes/class.filter-embedded-html-objects.php

```diff
@@ -230,7 +230,10 @@ public static function maybe_create_links( $string ) {
 
 		foreach ( self::$failed_embeds as $entry ) {
 			$html   = sprintf( '<a href="%s">%s</a>', esc_url( $entry['src'] ), esc_url( $entry['src'] ) );
-			$string = str_replace( $entry['match'], $html, $string );
+			// Check if the string doesn't contain iframe, before replace.
+			if ( ! preg_match('/<iframe /',$string) ) {
```

Comment:
Could you add some spaces inside the braces, as mentioned [here](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage)?

Thanks!